app/gps/intersect/toExposureInfo.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { DayBins, toExposureHistory } from './toExposureHistory';
+import { ExposureDatum } from '../../exposureHistory';
+import dayjs from 'dayjs';
+
+describe('toExposureHistory', () => {
+  describe('when given a 21 day calendar', () => {
+    describe('when given the DayBins with no exposures', () => {
+      it('it returns a 21 day history of NoData Exposures', () => {
+        const today = dayjs('2020-06-26').valueOf();
+        const rawExposures: DayBins = [];
+
+        const result = toExposureHistory(rawExposures, {
+          initDate: today,
+          totalDays: 21,
+        });
+
+        expect(result.length).toBe(21);
+        expect(
+          result.every((datum: ExposureDatum) => datum.kind === 'NoData'),
+        ).toBe(true);
+      });
+    });
+
+    describe('when given the DayBins with positive duration today and 2 days ago', () => {
+      it('returns an exposure history with the exposures on the correct days', () => {
+        const tuesday = dayjs('2020-06-23').valueOf();
+        const friday = dayjs('2020-06-26').valueOf();
+
+        const today = friday;
+
+        const dayBins: DayBins = [300000, -1, 0, 600000, 0];
```

Comment:
super helpful to see some data for this. as a reader not familiar with the gps flow, i find `DayBins` not super clear.

That said, what do you think typing this like

```typescript
type IntersectionDuration = number
type DayBin = IntersectionDuration[]

const dayBins: DayBin = [30000, -1, 0]
```

or something like that. it probably makes sense in a different commit, but in general, what exactly is a dayBin? and can we add more type safety here than a list of numbers?